src/modm/platform/gpio/stm32/module.lb

```diff
@@ -273,3 +273,7 @@ def build(env):
     env.template("unused.hpp.in")
     if len(env["enable_ports"]):
         env.template("enable.cpp.in")
+
+    # FIXME: Move to modm:platform:core!
+    env.outbasepath = "modm/src/modm/platform/core"
+    env.template("peripherals.hpp.in")
```

Comment:
modm-devices need to become smarter, right now it's just dumping the raw XML out it's API, so a lot of things need to be computed in modm that should be provided by modm-devices. Here, to compute the true set of addressable peripherals requires quite a bit of computation, which I don't want to duplicate in `:platform:core` module.